pkgs/development/tools/rgp/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, stdenv, makeWrapper, fetchurl, autoPatchelfHook,
+  dbus, fontconfig, freetype, glib, libglvnd, libX11, libxcb, libXi, ncurses5,
+  qtbase, zlib
+}:
+with stdenv.lib;
+
+let
+  deps = with pkgs; [
+    dbus
+    fontconfig
+    freetype
+    glib
+    libglvnd
+    libX11
+    libxcb
+    libXi
+    ncurses5
+    qtbase
+    zlib
+  ];
+
+in stdenv.mkDerivation rec {
+  pname = "rgp";
+  version = "1.7.0.29";
+
+  src = fetchurl {
+    url = https://github.com/GPUOpen-Tools/Radeon-GPUProfiler/releases/download/v1.7/RadeonGPUProfiler_1.7.0.29.tgz;
+    sha256 = "d7d4aa92231796a509dfcf92e9618cac98ae6572c12efd0871eb43afa4e8240a";
+  };
+
+  nativeBuildInputs = [ makeWrapper autoPatchelfHook ];
+
+  buildInputs = deps;
+
+  rpath = lib.makeLibraryPath (deps ++ [ stdenv.cc.cc.lib ]);
```

Comment:
Is `rpath` a standard attribute to `mkDeviation`? I'm not familiar with it. Where is this used? Can you point me to some documentation?
-

Reply:
Sorry, this was a leftover from trying to set the rpath, now this is done by `autoPatchelfHook`, so not needed anymore.